packages/tailwindcss/src/utilities.ts

```diff
@@ -4937,16 +4956,23 @@ export function createCssUtility(node: AtRule) {
       })
 
       designSystem.utilities.suggest(name.slice(0, -2), () => {
-        return [
-          {
-            values: designSystem.theme
-              .keysInNamespaces(valueThemeKeys)
-              .map((x) => x.replaceAll('_', '.')),
-            modifiers: designSystem.theme
-              .keysInNamespaces(modifierThemeKeys)
-              .map((x) => x.replaceAll('_', '.')),
-          },
-        ] satisfies SuggestionGroup[]
+        let values = []
+        for (let value of valueLiterals) {
+          values.push(value)
+        }
+        for (let value of designSystem.theme.keysInNamespaces(valueThemeKeys)) {
+          values.push(value.replaceAll('_', '.'))
```

Comment:
Didn't we drop the .replaceAll(…) thing elsewhere in the codebase or no? I forget… I just remember something about that changing at some point

Reply:
Yep is covered in the `suggest` function itself.